fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/NodePropertiesTools.java

```diff
@@ -213,18 +222,34 @@ public void removeNodeProperty(final Node node, final String propertyName, final
                 // we only need to update the property if we did anything.
                 if (remove.get()) {
                     if (newValues.length == 0) {
-                        LOGGER.debug("Removing property {}", propertyName);
+                        LOGGER.debug("Removing property '{}'", propertyName);
                         property.remove();
                     } else {
-                        LOGGER.debug("Removing value {} from property {}", valueToRemove, propertyName);
+                        LOGGER.debug("Removing value '{}' from property '{}'", valueToRemove, propertyName);
                         property.setValue(newValues);
                     }
+                } else {
+                    LOGGER.info("Value not removed from property name '{}' (value '{}')", propertyName,
+                            valueToRemove);
                 }
 
             } else {
-                if (property.getValue().equals(valueToRemove)) {
-                    LOGGER.debug("Removing value from property {}", propertyName);
-                    property.remove();
+
+                String strVal = "";
+                try {
+                    strVal = property.getValue().getString();
+
+                    if (property.getValue().equals(valueToRemove)) {
+                        LOGGER.debug("single value: Removing value from property {}", propertyName);
+                        property.remove();
+                    } else {
+                        LOGGER.info("Value not removed from property name '{}' (value '{}')", propertyName,
```

Comment:
I put in 'info', so that something might come out in the log, else it's a silent fail.  Suggestions on how to handle the 'else' in this case welcome.